.gitlab-ci.yml

```diff
@@ -47,7 +47,6 @@ test:lint:
       - screenshots
       - logs
       - junit
-    when: on_failure
```

Comment:
Is this a left-over, or intended change?

Reply:
That's intended to allow inspection of screenshots and replays even if a test was just flaky (or in this case passed, but the terminal diff would have been interesting).